android/src/main/java/com/amplitude/skylab/Skylab.java

```diff
@@ -48,11 +52,36 @@ public static SkylabClient getInstance() {
      * Returns the {@link SkylabClient} instance associated with the provided name. If no Client
      * was initialized with the given name, returns null.
      */
-    public static SkylabClient getInstance(String name) {
+    @Nullable
+    public static SkylabClient getInstance(@Nullable String name) {
         String normalizedName = SkylabConfig.normalizeInstanceName(name);
         return INSTANCES.get(normalizedName);
     }
 
+    /**
+     * Returns the default {@link SkylabClient} instance associated with the provided name.
+     * @throws IllegalStateException If the default client has not been initialized.
+     */
+    @NotNull
+    public static SkylabClient getInstanceOrThrow() {
```

Comment:
Some additional functionality to add a `@NotNull` value for people who are confident that the instance has been initialized.